src/main/java/net/javadiscord/javabot/util/MessageActionUtils.java

```diff
@@ -61,7 +61,7 @@ public static CompletableFuture<Message> addAttachmentsAndSend(Message message,
 		List<CompletableFuture<?>> attachmentFutures = new ArrayList<>();
 		for (Message.Attachment attachment : message.getAttachments()) {
 			attachmentFutures.add(
-					attachment.retrieveInputStream()
+					attachment.getProxy().download()
```

Comment:
Does this work with non-image attachments?

Reply:
yes